src/galaxis-footer/pages/pages.component.tsx

```diff
@@ -1,19 +1,38 @@
-import React, { FC } from "react";
+import React, { FC, useEffect, useState } from "react";
 import { PageModel } from "../../models/page.model";
-import { Box, Typography } from "@mui/material";
+import { Box, Link, Typography } from "@mui/material";
 import CustomTitle from "../title/title.component";
 import StyledWrapper from "./pages.style";
 
 type PagesProps = { pages: PageModel[] };
 
-const Pages: FC<PagesProps> = ({ pages }) => {
+const Pages: FC<PagesProps> = ({ pages: initialPages }) => {
+  const [pages, setPages] = useState<PageModel[]>(initialPages);
+
+  useEffect(() => {
+    const path = window.location.pathname;
+    if (path.includes("engines")) {
```

Comment:
The `engines` is an example in the ticket. We discussed to many of Apps will use this component.